quickwit-storage/src/object_storage/s3_compatible_storage.rs

```diff
@@ -605,13 +609,24 @@ mod tests {
         assert_eq!(split_range_into_chunks(0, 1), vec![]);
     }
 
-    use super::parse_split_uri;
+    use super::parse_uri;
 
     #[test]
-    fn test_parse_split_uri() {
+    fn test_parse_uri() {
+        assert_eq!(
```

Comment:
does  `parse_uri("s3://bucket")` is accepted ? Is it consistent with aws s3 ?

Reply:
@fmassot  I have relaxed the validation after testing